packages/@expo/config-types/src/ExpoConfig.ts

```diff
@@ -40,9 +40,10 @@ export interface ExpoConfig {
    * **Note: Don't use this property unless you are sure what you're doing**
    *
    * The runtime version associated with this manifest.
-   * Set this to `{"policy": "nativeVersion"}` to generate it automatically.
+   * Set this to `{"policy": "nativeVersion"}` to generate it automatically based on the 'version' and
+   * 'android.versionCode'/'ios.buildNumber' or to `{"policy": "appVersion"}` to use just 'version' field.
    */
-  runtimeVersion?: string | { policy: 'nativeVersion' | 'sdkVersion' };
+  runtimeVersion?: string | { policy: 'nativeVersion' | 'sdkVersion' | 'appVersion' };
```

Comment:
```suggestion
  runtimeVersion?: string | { policy: 'nativeVersion' | 'sdkVersion' | 'appVersion' };
```
Can we extract possible policy values to the separate type, and the re-use it in those fields?